pipeline.groovy

```diff
@@ -130,6 +130,16 @@ def getSources (name) {
     }
 }
 
+def test_travis_yaml () {
+    def script = 'dub test --compiler=$DC'
+    if (fileExists('.travis.yml')) {
+        def travis_script = sh(script: 'get_travis_test_script', returnStdout: true).trim()
```

Comment:
I couldn't find any reference to `get_travis_test_script`, so I assume it's a magic binary/script that's within the `PATH`?